packages/e2e/cypress/integration/ui/components/in-app-messaging/demo/demo.steps.ts

```diff
@@ -0,0 +1,31 @@
+import { Given, Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+Given('{string} checkbox is checked', (checkboxName: string) => {
+  cy.findByText(checkboxName)
```

Comment:
Wondering if we can use `cy.findByRole` if possible, since `findByRole` is better at keeping up accessibility standards in our lib.

Reply:
Tried to use `cy.findByRole` wherever possible. In this case, it was a span element used next to another span element being used as checkbox, so couldn't use `findByRole`.